src/tests/testClpMap.cc

```diff
@@ -254,7 +254,6 @@ void
 testClpMap::testPurgeIsLRU()
 {
     TestMap m(2048);
-    // TODO: once we have ClpMap iterators we can inspect the contents instead
```

Comment:
In my mind they would, because instead of keeping an external index and very carefully inspecting some elements, we could just look them up, and then do set comparison. Not a big deal tho

Reply:
I am not convinced because the code is already "looking them up", without iterators. Moreover, iterators are not really meant for looking up _specific_ values, which is what this code needs AFAICT. However, I would be happy to be proven wrong: We agree that this code needs improvement, and the details are not important at the moment.